src/app/app.component.ts

```diff
@@ -1,6 +1,5 @@
 import * as jq from 'jquery';
-import * as _swal from 'sweetalert';
-import { SweetAlert } from 'sweetalert/typings/core';
+import Swal from 'sweetalert2/src/sweetalert2.js';
```

Comment:
We format all imports like so: 

```
import { Swal } from 'sweetalert2/src/sweetalert2.js';
```

Reply:
sweetalert2 does not export a member named Swal, so this won't work like this, as its an entirely different import statement. (I tried and got TS2305)

Reply:
Kept unchanged as implied with comment.